ffi/SDL2_0.lua

```diff
@@ -493,4 +493,8 @@ function S.getPowerInfo()
     return batt, charging, plugged, percent
 end
 
+function S.getVersion()
+    return string.format("%d.%d.%d", sdl_linked_ver[0].major, sdl_linked_ver[0].minor, sdl_linked_ver[0].patch)
```

Comment:
Fine in principle, but I'm kind of wondering if it'd be nicer to add something along the lines of:
```lua
local function getSDLVersion()
    return sdl_linked_ver[0].major, sdl_linked_ver[0].minor, sdl_linked_ver[0].patch
end
```
https://github.com/koreader/koreader-base/blob/0cef37348757e755c47aeee7ef6f8c743e061d11/ffi/SDL2_0.lua#L30-L44

To use in both places.

Reply:
Done!